### PR TITLE
optionally ignore last block on resume

### PR DESCRIPTION
**Pull request by ssallam**
closes issue #392

### CODE REVIEW

File: aquarius/events/events_monitor.py [hunk 1]
```diff
@@ -91,9 +91,13 @@ def __init__(self, web3, config_file, metadata_contract=None):
         if self._ecies_private_key:
             self._ecies_account = Account.privateKeyToAccount(self._ecies_private_key)
 
+        ignore_last_block = get_bool_env_value("IGNORE_LAST_BLOCK", 0)
```
Comment (calina-c) [inline aquarius/events/events_monitor.py:94]: can we have unit tests and documentation for this option?

Comment (mariacarmina) [reply aquarius/events/events_monitor.py:94]: agreed.

**Comment by calina-c** [comment]
@mariacarmina please add a few unit tests for the get_bool_env_value function, and add the option to the docs for reference.

**Comment by calina-c** [comment]
Codacy fails because `self` is unused, but I have to keep the same method signature that is expected for the concrete classes. So there's not much I can do.

**Pull request merged**